packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -141,8 +141,8 @@ function removeIframe(iframe: HTMLIFrameElement) {
 
 interface BrowserEnvironmentOptions {
     distortionCallback?: DistortionCallback;
-    endowments?: object;
-    globalThis: WindowProxy & typeof globalThis;
+    endowments: object;
+    globalThis?: WindowProxy & typeof globalThis;
```

Comment:
`globalThis?:` shouldn't this be required still and not optional

Reply:
same as above.